test/test_functional_tensor.py

```diff
@@ -1141,5 +1003,148 @@ def test_adjust_gamma(device, dtype, config):
+
+    # scriptable function test
+    hflipped_img_script = script_hflip(img_tensor)
+    assert_equal(hflipped_img, hflipped_img_script)
+
+    batch_tensors = _create_data_batch(16, 18, num_samples=4, device=device)
+    _test_fn_on_batch(batch_tensors, F.hflip)
+
+
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('top,left,height,width', [(1, 2, 4, 5),   # crop inside top-left corner
+                                                   (2, 12, 3, 4),  # crop inside top-right corner
+                                                   (8, 3, 5, 6),   # crop inside bottom-left corner
+                                                   (8, 11, 4, 3),  # crop inside bottom-right corner
+                                                   ]
+                         )
+def test_crop(device, top, left, height, width):
+    script_crop = torch.jit.script(F.crop)
+
+    img_tensor, pil_img = _create_data(16, 18, device=device)
+
+    pil_img_cropped = F.crop(pil_img, top, left, height, width)
+
+    img_tensor_cropped = F.crop(img_tensor, top, left, height, width)
+    _assert_equal_tensor_to_pil(img_tensor_cropped, pil_img_cropped)
+
+    img_tensor_cropped = script_crop(img_tensor, top, left, height, width)
+    _assert_equal_tensor_to_pil(img_tensor_cropped, pil_img_cropped)
+
+    batch_tensors = _create_data_batch(16, 18, num_samples=4, device=device)
+    _test_fn_on_batch(batch_tensors, F.crop, top=top, left=left, height=height, width=width)
+
+
+@pytest.mark.parametrize('device', cpu_and_gpu())
+@pytest.mark.parametrize('image_size', ('small', 'large'))
+@pytest.mark.parametrize('dt', [None, torch.float32, torch.float64, torch.float16])
+@pytest.mark.parametrize('ksize', [(3, 3), [3, 5], (23, 23)])
+@pytest.mark.parametrize('sigma', [[0.5, 0.5], (0.5, 0.5), (0.8, 0.8), (1.7, 1.7)])
+@pytest.mark.parametrize('fn', [F.gaussian_blur, torch.jit.script(F.gaussian_blur)])
+def test_gaussian_blur(device, image_size, dt, ksize, sigma, fn):
+
+    # true_cv2_results = {
+    #     # np_img = np.arange(3 * 10 * 12, dtype="uint8").reshape((10, 12, 3))
+    #     # cv2.GaussianBlur(np_img, ksize=(3, 3), sigmaX=0.8)
+    #     "3_3_0.8": ...
+    #     # cv2.GaussianBlur(np_img, ksize=(3, 3), sigmaX=0.5)
+    #     "3_3_0.5": ...
+    #     # cv2.GaussianBlur(np_img, ksize=(3, 5), sigmaX=0.8)
+    #     "3_5_0.8": ...
+    #     # cv2.GaussianBlur(np_img, ksize=(3, 5), sigmaX=0.5)
+    #     "3_5_0.5": ...
+    #     # np_img2 = np.arange(26 * 28, dtype="uint8").reshape((26, 28))
+    #     # cv2.GaussianBlur(np_img2, ksize=(23, 23), sigmaX=1.7)
+    #     "23_23_1.7": ...
+    # }
+    p = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'assets', 'gaussian_blur_opencv_results.pt')
+    true_cv2_results = torch.load(p)
+
+    if image_size == 'small':
+
+        tensor = torch.from_numpy(
+            np.arange(3 * 10 * 12, dtype="uint8").reshape((10, 12, 3))
+        ).permute(2, 0, 1).to(device)
+
+    else:
```

Comment:
no need to skip lines here:

```suggestion
    if image_size == 'small':
        tensor = torch.from_numpy(
            np.arange(3 * 10 * 12, dtype="uint8").reshape((10, 12, 3))
        ).permute(2, 0, 1).to(device)
    else:
```